Formula/jack.rb

```diff
@@ -85,6 +72,31 @@ def plist
   end
 
   test do
-    assert_match version.to_s, shell_output("#{bin}/jackd --version")
+    source_name = "test_source"
+    sink_name = "test_sink"
+    jackd_cmd = "#{bin}/jackd -X coremidi -d dummy"
+    midi_source_cmd = "#{bin}/jack_midiseq #{source_name} 16000 0 60 8000"
+    midi_sink_cmd = "#{bin}/jack_midi_dump #{sink_name}"
+    midi_connect_cmd = "#{bin}/jack_connect #{source_name}:out #{sink_name}:input"
+    ohai jackd_cmd
+    jackd = IO.popen jackd_cmd
+    system "#{bin}/jack_wait", "--wait", "--timeout", "10"
+    ohai midi_source_cmd
+    midi_source = IO.popen midi_source_cmd
```

Comment:
This should be something like 

```
Fork do
  system "cmd"
End
```

Reply:
My amateur implementation of this concept results in tests failing. @ColinKinloch do you have any ways to remedy this?

Reply:
The solution was to keep the `IO.popen` corresponding to the `jack_sink_cmd` as is.